tools/scylla-sstable.cc

```diff
@@ -452,7 +452,7 @@ class dumping_consumer : public sstable_consumer {
                 }
                 _writer.EndArray();
             } else {
-                _writer.String("<unknown>");
+                _writer.Null();
```

Comment:
null is unique but not very distinct. For something illegal in the input stream, we should have  (in addition) some extra key/value pair illegal: true or illegal: "error message" so it stands out more.

Reply:
In general I agree, but here null is distinct and easy to check for as normally the value here is an object.

Reply:
Actually an error message would be nice here, but this patch is a "shouldn't-happen" style undefined behavior which I wanted to make defined. Maybe I should have just added an assert to the code. But I'm probably over-thinking this for a simple tool like this.